### DISCUSSION TITLE
Mass storage mounting

### DISCUSSION THREAD

**Discussion by croadfeldt** [tiny-pilot/tinypilot#513] [Q&A] [answered]
First off, thanks for spinning this tool up!
Seoncd, pardon my ignorance if this has already been discussed.

I've got a working solution for mouting disk images as removable drives via the OTG USB connection. It works and I can add / remove images as needed live.

Is there any reason why this ability has not been added prior? Once I get done with the scripts to automate the process, will send over a PR for it. But wanted to see if there was a gotcha I'm missing before I do.

**Comment by mtlynch** [comment, marked as answer]
The hard part is really creating the friendly UI and robust code around all the gotchas. I've had a working prototype for months, but the hard part is making it resilient and user-friendly.

I expect that we'll add this feature in the next 4-6 weeks, but it will be part of the [Pro package](https://tinypilotkvm.com/product/tinypilot-pro) rather than the free community version.
